README.md

```diff
@@ -57,6 +57,19 @@ val driver: SqlDriver = JdbcSqliteDriver(IN_MEMORY)
 Database.Schema.create(driver)
 ```
 
+#### Javascript
+```groovy
+dependencies {
+  implementation "com.squareup.sqldelight:sqljs-driver:1.3.0"
+}
+```
+```kotlin
+val sql = initSql()
+val db = sql.Database()
+val driver: SqlDriver = JsSqlDriver(db)
```

Comment:
can the constructor for `JsSqlDriver()` just `initSql` and get the database itself instead of doing these three steps explicitly? Or is the expectation that consumers would use the results of `initSql` and `Database()` multiple times

Reply:
I added some helper functions to initialize the driver directly